config/routes.rb

```diff
@@ -19,13 +23,11 @@
   # mandrill-rails
   resource :inbox, controller: "inbox", only: [:show, :create]
 
-  namespace :api do
-    scope :v1 do
-      namespace :ncr do
-        resources :work_orders, only: [:index]
+  if ENV["API_ENABLED"] == "true"
+    namespace :api do
+      scope :v2 do
```

Comment:
I think it makes sense to namespace the version, but do we need to make this v2? afaik nobody uses v1